### PR TITLE
Add interactive checks on entering the password

### PR DESCRIPTION
**Pull request by wpears**
Closes https://github.com/cfpb/hmda-platform-auth/issues/120

### CODE REVIEW

**Comment by wpears** [comment]
@awolfe76 ✔️ it out

File: keycloak/themes/hmda/login/register.ftl [hunk 1]
```diff
@@ -39,12 +39,36 @@
...
+                  <img class="check" src="${url.resourcesPath}/img/correct9.png">
+                  <img class="missing" src="${url.resourcesPath}/img/close.png">
+                  Be at least 12 characters
+                </li>
+                <li data-validator="uppercase">
+                  <img class="check" src="${url.resourcesPath}/img/correct9.png">
+                  <img class="missing" src="${url.resourcesPath}/img/close.png">
+                  Have at least 1 uppercase character
+                </li>
+                <li data-validator="lowercase">
```
Comment (awolfe76) [inline keycloak/themes/hmda/login/register.ftl]: Is it possible to use `background` on the `li` to update the icons? Then we could remove the `` tags from here.

So, `<li class="missing">` could get a style of `background: url('theX.png') no-repeat ...`.

File: keycloak/themes/hmda/login/resources/css/hmda.css [hunk 2]
```diff
@@ -163,6 +163,31 @@ body {
...
+#validation_list img {
+  display: none;
+  position: absolute;
+  height: 18px;
+  width: 18px;
+  left: -22px;
+  bottom: 5px;
+}
+
+#validation_list > li.complete > .check {
```
Comment (awolfe76) [inline keycloak/themes/hmda/login/resources/css/hmda.css]: (See comment above)

~~I think you could just do `#validation_list > li.complete > img` for this rule and the one below it (`#validation_list > li.missing > .missing`). That should take care of both.~~

~~You could also separate them with a `,`, like you've done [above](https://github.com/cfpb/hmda-platform-auth/pull/165/files#diff-e1f62c956c8a0bbd4eeb2efb7debcbafR170).~~

~~Then you could remove the classes on the images, [for example](https://github.com/cfpb/hmda-platform-auth/pull/165/files#diff-0d19d4c8d9acc4f8c775f35627a24d52R43).~~

File: keycloak/themes/hmda/login/register.ftl (hunk 1 above)
Comment (awolfe76) [reply keycloak/themes/hmda/login/register.ftl]: Ahh, I see. I think this ok. 👍 

Maybe we can investigate it later.

**Pull request merged**